src/actions/action-creators.test.js

```diff
@@ -126,7 +128,147 @@ describe('Action creators', () => {
     });
   });
 
+  /***** async *****/
   describe('async action creators', () => {
-    // TODO
+    let store;
+    beforeEach(() => {
+      store = mockStore({
```

Comment:
i'm probably move this object into an `initialState` constant at the top of the file.

Reply:
Done